OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/StatisticControllerTests.cs

```diff
@@ -30,60 +30,72 @@ public void SetUp()
         }
 
         [Test]
-        [TestCase(5)]
-        [TestCase(2)]
-        [TestCase(12)]
-        public async Task GetWorkshops_WhenLimitIsValid_ShouldReturnOkResultObject(int limit)
+        [TestCase(5, "")]
+        [TestCase(2, "")]
+        [TestCase(12, "")]
+        public async Task GetWorkshops_WhenLimitIsValid_NoCity_ShouldReturnOkResultObject(int limit, string city)
```

Comment:
What does `NoCity` actually means ?
also `LimitIsValid` means that the  value is from the valid value range ?
If you want to use the value from your existing data - use collection you're set up for that case.